README.md

```diff
@@ -1,15 +1,59 @@
 # hello-hydra
 
-An example eosio.token contract tested with [Hydra](docs.klevoya.com/hydra).
+[![N|Solid](https://cldup.com/dTxpPi9lDf.thumb.png)](https://nodesource.com/products/nsolid)
 
-To install the dependencies run:
+[![Build Status](https://travis-ci.org/joemccann/dillinger.svg?branch=master)](https://travis-ci.org/joemccann/dillinger)
+
+Hydra is an EOSIO smart contract testing framework that allows you to quickly get started with testing your smart contract code. Besides Hydra, installing additional software or running your own local blockchain node is not required. 
+
+# New Features!
+
+  - Import a HTML file and watch it magically convert to Markdown
+
+You can also:
+  - Import and save files from GitHub, Dropbox, Google Drive and One Drive
+
+### Tech
+
+Hydra uses a number of open source projects to work properly:
+
+* [Jest] - a delightful javascript testing framework maintained by facebook with a focus on simplicity
+
+### Installation
+
+Hydra requires [Node.js](https://nodejs.org/) v10.16.0 to run.
+
+Install the dependencies and devDependencies and sign up or login with your existing Github or Google account
+
+```sh
+$ git clone https://github.com/klevoya/hello-hydra.git
+$ cd hello-hydra/
+$ sudo chmod +x setup.sh
+$ sudo bash ./setup.sh
```

Comment:
Imo, the script is a bit too agressive. We shouldn't just uninstall the user's node version and install nvm and use node 10.

Especially, as we don't mention it here and it looks like `setup.sh` is required for any Hydra users. I think a lot of devs already have node installed.

Reply:
The script also only works on Ubuntu / distros with `apt`. Everyone else will see an error when running the script and might think that Hydra doesn't work for them.

I don't think we need the script to be honest. Refering to the official Nodejs installation guide should be enough in my opinion.